README.asciidoc

```diff
@@ -441,6 +441,41 @@ docker {
 }
 ----
 
+[TIP]
```

Comment:
How this is formatted is kind of odd IMO. Can we just do a typical "section" like we have elsewhere? I think that would suffice.